README.md

```diff
@@ -59,30 +59,44 @@ A `.dmg` should appear in `build/`
 
 ### Updating patches
 
+1. Start the process and set the environment variables
 ```sh
 ./devutils/update_patches.sh merge
 source devutils/set_quilt_vars.sh
+```
 
-# Setup Chromium source
+2. Setup Chromium source
+```sh
 mkdir -p build/{src,download_cache}
 ./ungoogled-chromium/utils/downloads.py retrieve -i ungoogled-chromium/downloads.ini downloads.ini -c build/download_cache
 ./ungoogled-chromium/utils/downloads.py unpack -i ungoogled-chromium/downloads.ini downloads.ini -c build/download_cache build/src
-
 cd build/src
-# Use quilt to refresh patches. See ungoogled-chromium's docs/developing.md section "Updating patches" for more details, up until "quilt pop -a"
-quilt pop -a
-cd ..
-# Validate that patches are applied correctly
-./ungoogled-chromium/devutils/validate_patches.py -l build/src -p patches -s patches/series
+```
+
+3. Use `quilt` to refresh all patches: `while quilt push; do quilt refresh; done`
+	* If an error occurs, go to the next step. Otherwise, skip to Step 5.
+4. Use `quilt` to fix the broken patch:
+    1. Run `quilt push -f`
+    2. Edit the broken files as necessary by adding (`quilt edit ...` or `quilt add ...`) or removing (`quilt remove ...`) files as necessary
+        * When removing large chunks of code, remove each line instead of using language features to hide or remove the code. This makes the patches less susceptible to breakages when using quilt's refresh command (e.g. quilt refresh updates the line numbers based on the patch context, so it's possible for new but desirable code in the middle of the block comment to be excluded.). It also helps with readability when someone wants to see the changes made based on the patch alone.
+    3. Refresh the patch: `quilt refresh`
+    4. Go back to Step 3.
+5. Run `../../ungoogled-chromium/devutils/validate_config.py`
+6. Run `quilt pop -a`
+
+7. Validate that patches are applied correctly
+```sh
+cd ../..
+./ungoogled-chromium/devutils/validate_patches.py -l build/src
```

Comment:
This should be updated to include arguments `-p` and `-s` as specified above. Otherwise, the patches and series in the ungoogled-chromium repo will be used instead. I may change that script so it will try files in the current working directory instead of files relative to the script.

Reply:
I've updated `validate_patches.py` in a new commit that I will tag soon: https://github.com/Eloston/ungoogled-chromium/commit/43ec829d19d655b879ada05137cac65aab73581e

Reply:
I could update ungoogled-chromium to the new tag and refresh the patches turning this PR in an update to 136. Is it ok if i hold back until they release it officially?
Right now, the Chrome that is being downloaded is still 131.

Reply:
Yeah, we can hold this PR until 136 is merged, or you can just update the arguments for now (they won't break after updating to 136, but they will become unnecessary)

Reply:
Done, also fixed validation since we have series.merged at that point.